tasks/deploy_get_stamp_from_route.yml

```diff
@@ -29,12 +29,33 @@
     verbosity: 3
   when: app_route is defined
 
-- name: set_fact app_route_deployment_stamp
+- name: Get service targeted
   set_fact:
-    app_route_deployment_stamp: "{{ app_route.ansible_facts.route.metadata.labels.deployment_stamp | default(none) }}"
+    targeted_service: |
+      {{
+        lookup(
+          'openshift',
+          api_version='v1',
+          namespace=project_name,
+          kind='Service',
+          resource_name=app_route.ansible_facts.route.spec.to.name
```

Comment:
I think this query may return a 404 response when the current and/or previous routes point to a non-existing service (this is the default after the project's initialization).

Reply:
it returns an empty list. You can test it the task after, the var is printed.

Reply:
Do you think the k8s client behavior is different when running python 3?